express-importer/pom.xml

```diff
@@ -0,0 +1,172 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.axonivy.portal.util</groupId>
```

Comment:
this is the namespace of portal

Reply:
I added the "util" part? But I'm open for a better ID . Any suggestions?

Reply:
all new portal projects:
<groupId>com.axonivy.portal</groupId>

doc-factory
  <groupId>com.axonivy.utils.docfactory</groupId>


proposal
<groupId>com.axonivy.portal.express</groupId>

someday. express and express importer should live in it's same repository. there is a high coupling.

maybe it should not have portal in the namepsace. but then we can change both.